### DISCUSSION TITLE
preparation steps

### DISCUSSION THREAD

**Discussion by Joe-ncB** [nextcloud/cookbook#1233] [FAQ and user questions]
when in add a recipe automatically i might have 7 or 8 steps i can see
when i add one manually i can do this as well
BUT after saving and opening again i found that there are only 6 steps saved
the rest is cut off

am i missing something or is this a problem wich has to be reviewed ??

**Comment by christianlupus** [comment]
You mean that when adding a recipe manually, you cannot add more than 6 instructions/ingredients permanently? This is not intended. You should be able to add 100 entries of you like. 

Please open a bug report and ideally attach a recipe example (the json file) as additional information.

**Comment by Joe-ncB** [comment]
such a bad example ...

now all is fine (i dont understand)

must be something else -- last night i had this behavior 3 times

i keep an eye on it

sorry for inconvinience

**Reply by christianlupus** [reply]
All right, no problem.
